mack/__init__.py

```diff
@@ -105,6 +108,47 @@ def kill_duplicates(deltaTable, pkey, cols):
     ).whenMatchedDelete().execute()
 
 
+def drop_duplicates(delta_table: DeltaTable, primary_key: str, duplication_columns: List[str]):
+    if not delta_table:
+        raise Exception("An existing delta table must be specified.")
+
+    if not primary_key:
+        raise Exception("A primary key must be specified.")
+
+    if not duplication_columns or len(duplication_columns) == 0:
```

Comment:
can the `primary_key` be the only column in `duplication_columns`? we should probably prevent that

Reply:
Do you mean like `mack.drop_duplicates(deltaTable, "col1", ["col1"])`? I will check that duplication_columns and primary_key are mutually exclusive

Reply:
Yup that's what I mean!